cmd/collectinfo.go

```diff
@@ -33,7 +33,9 @@ var collectinfoCmd = &cobra.Command{
 	Use:   "collectinfo",
 	Short: "collectinfo command collects all the required info from kubernetes cluster",
 	Long: `This command collects the following data from the given namespaces:
-* pods, statefulsets, persistentvolumeclaims, aerospikeclusters, nodes, storageclasses objects.
+* pods, statefulsets, deployments, persistentvolumeclaims, aerospikeclusters, aerospikebackupservices, 
+* aerospikebackups, aerospikerestores, configmaps, replicasets, poddisruptionbudgets, services,
+* nodes, storageclasses, persistentvolumes objects.
```

Comment:
See if we can arrange this info into Namespace and Cluster scoped objects for better readability

Reply:
Is this okay?
This command collects the following data from the given namespaces:
* pods, statefulsets, deployments, persistentvolumeclaims, aerospikeclusters, aerospikebackupservices, 
* aerospikebackups, aerospikerestores, configmaps, poddisruptionbudgets, service objects.

Collects the following data from the cluster:
* nodes, storageclasses, persistentvolumes objects.
* mutating and validating webhooks configurations.

and also collects the containers logs and events logs.